shared/chat/audio/audio-player.tsx

```diff
@@ -62,73 +68,57 @@ type Props = {
 }
 
 const AudioPlayer = (props: Props) => {
+  const {duration, big, maxWidth, url, visAmps} = props
   const seekRef = React.useRef<null | ((n: number) => void)>(null)
-  const [timeStart, setTimeStart] = React.useState(0)
-  const [timeLeft, setTimeLeft] = React.useState(props.duration)
-  const [timePaused, setTimePaused] = React.useState(0)
-  const [timePauseButton, setTimePauseButton] = React.useState(0)
+  const [playedRatio, setPlayedRatio] = React.useState(0)
   const [paused, setPaused] = React.useState(true)
   const onClick = () => {
     if (paused) {
-      if (timeStart === 0) {
-        setTimeStart(Date.now())
-      } else if (timePauseButton > 0) {
-        setTimePaused(timePaused + (Date.now() - timePauseButton))
-      }
       setPaused(false)
-      setTimePauseButton(0)
     } else {
-      setTimePauseButton(Date.now())
       setPaused(true)
     }
   }
-  React.useEffect(() => {
-    if (paused) {
-      return
-    }
-    const timer = setTimeout(() => {
```

Comment:
this can get very out of sync on android where the timers are really laggy. better to just plumb back the progress